### DISCUSSION TITLE
Trouble enabling Rosetta on m2 macbook pro and colima version 0.7.5

### DISCUSSION THREAD

**Discussion by whelanp** [abiosoft/colima#1149] [Q&A] [answered]
I installed using brew colima version 0.7.5

When editing the configuration
`colima start --edit
`
I enabled the following options
```
vmType: vz
rosetta: true
mountType: virtiofs

```

However starting I can see the following in the logs

> [hostagent] Starting VZ (hint: to watch the boot progress, see "/Users/user/.colima/_lima/colima/serial*.log")
> [hostagent] Setting up Rosetta share
> [hostagent] Unable to configure Rosetta: failed to create a new rosetta directory share caching option: unsupported build target macOS version for 14.0 (the binary was built with __MAC_OS_X_VERSION_MAX_ALLOWED=130300; needs recompilation)
> SSH Local Port: 49558
> [hostagent] Waiting for the essential requirement 1 of 2: "ssh"
> [hostagent] [VZ] - vm state change: running

I am using Sonoma 14.7 (23H124) - what should I do to get support for Rosetta?

Thanks

**Comment by abiosoft** [comment, marked as answer]
This is the first time I am seeing this error. 
Can you try installing the development version and see if the error message persists? 


```
brew uninstall colima
brew install --head colima
```

**Reply by whelanp** [reply]
Works nicely with the head version - thank you.

**Reply by jiaming10** [reply]
I still face the same problem, and have reverted to using qemu.

I'm using M2 Air, Sonoma 14.7.1.

I tried using `brew install --head colima`, which installs `colima version HEAD-ad5102e` at the point of writing.

Do you need any additional info that will help in troubleshoot?
